src/services/analytics/events/recovery.ts

```diff
@@ -0,0 +1,62 @@
+import { EventType } from '@/services/analytics'
+
+// These are used for the generic stepper flow events (Next, Back)
+export const SETUP_RECOVERY_CATEGORY = 'setup-recovery'
+export const START_RECOVERY_CATEGORY = 'start-recovery'
+export const REMOVE_RECOVERY_CATEGORY = 'remove-recovery'
+export const CANCEL_RECOVERY_CATEGORY = 'cancel-recovery'
+
+const RECOVERY_CATEGORY = 'recovery'
+
+export const RECOVERY_EVENTS = {
+  SETUP_RECOVERY: {
+    action: 'Set up recovery',
```

Comment:
`Enable recovery`?

Reply:
Current suggestion is to update it to "Start recovery set-up" once the new modal is implemented so I would leave it as it is for now.